lib/dal/src/audit_logging.rs

```diff
@@ -88,6 +88,13 @@ pub(crate) async fn publish_pending(
         None => (TaskTracker::new(), true),
     };
 
+    // TODO(nick): somewhere in this function (or alongside it), we need to deserialize the events and send them up to
+    // the frontend. The frontend can make its own decision to include it into its paginated and filtered view, but we
+    // will make one event that sends an array of published audit logs. For chunking, we can do a naive array list
+    // length check or something else (e.g. size check) and send multiple messages, if need be. The first move might be
+    // to add a broadcast channel were shuttle sends the message paylods in bytestream format. Then, after shuttle
+    // exits, the dal can deserialize those messages.
```

Comment:
This is the next bit of work: getting rid of the refresh icon in the dashboard and publishing `WsEvents` for the newly published audit logs.